Classes/CodeGeneration/Extension/ExtensionGenerator.php

```diff
@@ -150,39 +149,24 @@ public function generate()
                     $controllerFolder
                 );
             }
-            if (true === $hasFluidpages) {
+            if ($this->configuration['pages'] ?? false) {
                 $this->appendControllerClassFile(
                     $filesToBeWritten,
                     'Page',
-                    'FluidTYPO3\\Fluidpages\\Controller\\PageController',
+                    'FluidTYPO3\\Flux\\Controller\\PageController',
                     $controllerFolder
                 );
             }
         }
         $this->appendTypoScriptConfiguration($filesToBeWritten);
         $this->appendExtensionLocalconfFile($filesToBeWritten);
-        if (true === $hasFluidcontent || true === $hasFluidpages) {
-            array_push($foldersToBeCreated, $this->targetFolder . '/Resources/Private/Language');
-        }
         $foldersToBeCreated = array_unique($foldersToBeCreated);
         foreach ($foldersToBeCreated as $folderPathToBeCreated) {
             $this->createFolder($folderPathToBeCreated);
         }
         foreach ($filesToBeWritten as $fileToBeWritten => $fileContentToBeWritten) {
             $this->createFile($fileToBeWritten, $fileContentToBeWritten);
         }
-        if (true === $hasFluidpages) {
```

Comment:
This part should not be removed - Flux supports the page and content templates as well as their associated icons, so even though the dependencies are no longer wanted, the icons still are ;)

Reply:
Good point :+1: